packages/@aws-cdk/core/test/@aws-solutions-konstruk/foo/package.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Do we really need all of this here? I think we can just do with a tiny `package.json` and `index.js` files.